pkg/driver/controller.go

```diff
@@ -77,7 +77,7 @@ func NewControllerService(c cloud.Cloud, o *Options) *ControllerService {
 }
 
 func (d *ControllerService) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
-	klog.V(4).InfoS("CreateVolume: called", "args", *req)
+	klog.V(4).InfoS("CreateVolume: called", "args", util.ClearSecrets(req))
```

Comment:
very minor np: maybe name this to something more generic like `SanitizeRequest`?

Reply:
Great suggestion, done ✅